lib/ex_aliyun_sls/client.ex

```diff
@@ -14,10 +14,11 @@ defmodule ExAliyunSls.Client do
   plug(Tesla.Middleware.Retry, delay: 5_000, max_retries: 5)
 
   def push2log_store(log_items, log_tags, topic, source, profile) do
-    %LogGroup{Logs: log_items, Source: source, LogTags: log_tags, Topic: topic}
-    |> LogGroup.encode!()
-    |> :erlang.iolist_to_binary()
-    |> request_api(profile)
+    {iodata, _size} = %LogGroup{Logs: log_items,
+                                Source: source,
+                                LogTags: log_tags,
+                                Topic: topic} |> LogGroup.encode!()
+    iodata |> request_api(profile)
```

Comment:
This modification forgot to call function `:erlang.iolist_to_binary/1`, resulting in the request body was not being converted to binary as expectation, I fixed it via https://github.com/edragonconnect/ex_aliyun_sls/commit/0bc7548e3fa6c10eae90e5e6153ae588c8acdb10, cc @gilbertwong96

Reply:
Hey, @xinz I didn't forget it.
<img width="1468" alt="image" src="https://github.com/user-attachments/assets/3e199072-d968-42bf-93da-6a5f57914937" />

As the code shows above, the body which is `iodata()` type can be handled correctly by the `:crypto.hash/2` function. 🤣

Reply:
But indeed I forgot to calculate the `body` length. Yes, your commit makes more sense.

Reply:
I just opened a new PR #3 to improve the details in https://github.com/edragonconnect/ex_aliyun_sls/commit/0bc7548e3fa6c10eae90e5e6153ae588c8acdb10